core/src/main/java/com/scalar/db/storage/jdbc/JdbcConfig.java

```diff
@@ -88,6 +89,7 @@ public JdbcConfig(DatabaseConfig databaseConfig) {
       throw new IllegalArgumentException(DatabaseConfig.CONTACT_POINTS + " is empty");
     }
     jdbcUrl = databaseConfig.getContactPoints().get(0);
+    jdbcDriver = databaseConfig.getJdbcDriver().orElse(null);
```

Comment:
auto-set for SQLite, or confirm why only SQLite needs to set JDBC Driver explicitly